actors/builtin/paych/paych_actor.go

```diff
@@ -172,6 +172,10 @@ func (pca Actor) UpdateChannelState(rt vmr.Runtime, params *UpdateChannelStatePa
 		rt.Abortf(exitcode.ErrIllegalArgument, "this voucher has expired!")
 	}
 
+	if sv.Amount.Sign() <= 0 {
+		rt.Abortf(exitcode.ErrIllegalArgument, "voucher amount not positive: %v", sv.Amount)
```

Comment:
This is also implicitly checked later, but it's still nice to implicitly check up-front.